eva-accession-import-automation/create_fasta_from_assembly_report.sh

```diff
@@ -95,15 +95,15 @@ do
                     # Uncompress file
                     mv ${output_folder}/${genbank_contig} ${output_folder}/${genbank_contig}.gz
                     gunzip ${output_folder}/${genbank_contig}.gz
-                    # Delete the accessions prefix
-                    sed -i 's/ENA|.*|//g' ${output_folder}/${genbank_contig}
                 fi
+                # Delete the accessions prefix
+                sed -i 's/ENA|.*|//g' ${output_folder}/${genbank_contig}
```

Comment:
Going into ultra-paranoid mode now that this script has been arising from the dead many times: 
I am suggesting a guard against white-spaces in the accessions line. This way, any brain-damaged input with spaces can be converted to the desired output.

![image](https://user-images.githubusercontent.com/25152678/53818245-b0abc180-3f5f-11e9-87fd-0c458701612f.png)